### PR TITLE
Profile list batch delete feature

### PR DESCRIPTION
**Pull request by TchaikovDriver**
Due to the inconvenience of profile deletion, now delete buttons in ServerListFragment are removed. Instead, batch deletion feature is implemented. Now it is much simpler to delete all servers and import new configuration.

### CODE REVIEW

File: app/src/main/java/io/github/trojan_gfw/igniter/servers/fragment/ServerListAdapter.java [hunk 1]
```diff
@@ -95,7 +95,7 @@ public void onClick(View v) {
 
     public void bindData(TrojanConfig config) {
         this.mConfig = config;
-        mRemoteAddrTv.setText(config.getRemoteAddr());
+        mRemoteAddrTv.setText(config.getRemoteServerRemark());
```
Comment (wongsyrone) [inline app/src/main/java/io/github/trojan_gfw/igniter/servers/fragment/ServerListAdapter.java:130]: How about renaming mRemoteAddrTv to something like mRemoteServerRemarkTv

Comment (TchaikovDriver) [reply app/src/main/java/io/github/trojan_gfw/igniter/servers/fragment/ServerListAdapter.java:134]: Yes, you are right.

Comment (TchaikovDriver) [reply app/src/main/java/io/github/trojan_gfw/igniter/servers/fragment/ServerListAdapter.java]: Done

**Comment by TchaikovDriver** [comment]
> image
> 
> Is this better than only displaying the remark?

I think remark is enough for choosing which server you want to connect to. Also, input field in MainActivity would remind user to reimport their configuration. For insurance, I will make a commit to display the remote address when remark is empty.

**Comment by who7708** [comment]
> > image
> > Is this better than only displaying the remark?
> 
> I think remark is enough for choosing which server you want to connect to. Also, input field in MainActivity would remind user to reimport their configuration. For insurance, I will make a commit to display the remote address when remark is empty.

Okay.

**Pull request merged**